### PR TITLE
Post edit delete articles

### PR DESCRIPTION
**Pull request by aiyus2020**

### CODE REVIEW

File: controllers/postArticleController.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+const {
+  newArticle,
+  updateArticleQuery,
+  deleteArticleQuery,
+} = require("../queries/articleQuery");
+class PostController {
+  async postArticle(req, res) {
+    try {
+      const { title, article } = req.body;
+      const userId = req.user;
```
Comment (courage173) [inline controllers/postArticleController.js:11]: this is still wrong. i thought I asked you to correct this. it should be req.user.id, you should always append the full user object to the req.user inside the auth middleware

File: models/dbMigration.js [hunk 2]
```diff
@@ -9,7 +9,9 @@ const client = new Client({
 
 async function dbMigration() {
   try {
-    await client.connect(); // Connect to the database
+    // Connect to the database
+    await client.connect();
+    console.log("Connected");
```
Comment (courage173) [inline models/dbMigration.js:14]: console

**Review by courage173 — CHANGES_REQUESTED** [review]

File: controllers/postArticleController.js (hunk 1 above)
Comment (aiyus2020) [reply controllers/postArticleController.js:11]: noted sir❤

File: models/dbMigration.js (hunk 2 above)
Comment (aiyus2020) [reply models/dbMigration.js:14]: noted

File: models/db.sql [hunk 3]
```diff
@@ -18,3 +18,10 @@ CREATE TABLE upload (
   title VARCHAR(225),
   image_url VARCHAR(225)
 );
+CREATE TABLE  articles (
+        id SERIAL PRIMARY KEY, 
+        user_id INTEGER NOT NULL REFERENCES user(id),
```
Comment (courage173) [inline models/db.sql:23]: it should be users

Comment (aiyus2020) [reply models/db.sql]: done

**Review by courage173 — APPROVED** [review]
Looks good

**Pull request merged**